proto/osmosis/txfees/v1beta1/query.proto

```diff
@@ -5,7 +5,7 @@ import "gogoproto/gogo.proto";
 import "google/api/annotations.proto";
 import "google/protobuf/duration.proto";
 
-import "osmosis/txfees/v1beta1/feetoken.proto";
```

Comment:
Was this done by a linter?

Reply:
I don’t know and of course that is a problem.